docs/usermanual.md

```diff
@@ -52,10 +52,11 @@ query parameters:
 * **t (timestamp)**: Timestamp of the measure. Will override the automatic IoTAgent timestamp (optional).
 
 #### Commands
+MQTT devices commands are always push. For HTTP Devices commands be push they **must** be provisioned with the `endpoint` attribute, that will contain
```

Comment:
Suggested alternative wording:

```
MQTT devices commands are always push. For HTTP Devices commands to be push they **must** be provisioned with the `endpoint` attribute, that will contain the URL where the IoT Agent will send the received commands. Otherwise the command will be poll.
```